src/brsTypes/components/ComponentFactory.ts

```diff
@@ -72,4 +95,20 @@ export class ComponentFactory {
                 return;
         }
     }
+
+    /**
+     * Checks to see if teh given component type can be resolved by the Factory
+     * That is, if it is a built in type or has been added at run time.
+     *
+     * @static
+     * @param {(BrsComponentName | string)} componentType
+     * @returns {boolean}
+     * @memberof ComponentFactory
```

Comment:
Like the others, some of these properties aren't necessary.  At the same time, I'd really appreciate seeing what those parameters and the return value represent!  Consider something like:

```suggestion
     * @param componentType - the component that might be resolvable
     * @returns - `true` if `componentType` can be resolved, otherwise `false`
```